src/block/block_write.c

```diff
@@ -235,7 +235,7 @@ __wt_block_write(WT_SESSION_IMPL *session, WT_BLOCK *block,
 int
 __wt_block_write_off(WT_SESSION_IMPL *session, WT_BLOCK *block,
     WT_ITEM *buf, wt_off_t *offsetp, uint32_t *sizep, uint32_t *cksump,
-    bool data_cksum, bool caller_locked)
+    bool data_cksum, bool checkpoint_io, bool caller_locked)
```

Comment:
Three bool args in a row is beyond my threshold for converting it into a `flags` arg.  (My threshold is two so both `wt_block_write` and `wt_block_write_off` could use flags, IMO.)